api/api.ts

```diff
@@ -41,7 +41,10 @@ app.post('/upload/:tenant', async (req: express.Request, res: express.Response)
 
         const filename: string = getFilename(req)
         const tenantConfig = await getDocByTenant(req.params.tenant)
-        const uploadJsonResponse = await uploadFile(tenantConfig, req.body, filename)
+
+        initializeDriver(tenantConfig)
+
+        const uploadJsonResponse = await uploadFile(tenantConfig, req.body)
```

Comment:
You don't need send `tenantConfig` here, just the raw bytes and the name of file.

```suggestion
        const uploadJsonResponse = await uploadFile(req.body, filename)
```

Reply:
done